.github/scripts/check-pr-text.py

```diff
@@ -41,7 +41,7 @@ def fail(message):
 if not pr_body:
     fail("Body for the PR not found. " "Maybe missing PR_BODY env var.")
 
-title_search = re.search(r"^(#\d+) .+", pr_title)
+title_search = re.search(r"^(#\d+):? .+", pr_title)
```

Comment:
I think that we should also update the PR template to document that multiple title formats are allowed.

AFAIK, the PR template is the only documenting for our PR  naming convention.

Reply:
:shrug: I think that a bot should be maintaining this invariant, not human beings.